TCAT/Controllers/CustomNavigationController.swift

```diff
@@ -139,25 +105,9 @@ class CustomNavigationController: UINavigationController, UINavigationController
         _ = popViewController(animated: true)
     }
 
-    @objc func reachabilityChanged(notif: Notification) {
-        if let reachability = notif.object as? Reachability {
-            switch reachability.connection {
-            case .wifi, .cellular:
-                banner.dismiss()
-            case .none:
-                banner.show(queuePosition: .front, on: self)
-                banner.autoDismiss = false
-                banner.isUserInteractionEnabled = false
-            }
```

Comment:
Are we not showing the banner anymore? I don't see that this code was moved anywhere, just deleted

Reply:
It's back now 😬